sdks/java/io/clickhouse/src/main/java/org/apache/beam/sdk/io/clickhouse/TableSchema.java

```diff
@@ -164,7 +166,8 @@ public enum TypeName {
     UINT32,
     UINT64,
     // Composite types
-    ARRAY
+    ARRAY,
```

Comment:
BOOL is not Composite type. Consider fix the comment here? Or is there potential breaking change if we put BOOL before ARRAY?

Reply:
I will fixed it probably missed that 
Thanks for the feedback

Reply:
Fixed the comment.